database/migrations/2020_02_21_230701_add_experience_personal_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddExperiencePersonalTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('experience_personal', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->string('title');
+            $table->longText('description');
+            $table->boolean('is_shareable');
+            $table->boolean('is_active')->nullable();
```

Comment:
Setting default(false) instead of nullable goes for all these migrations, but also for is_shareable in this case.